.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,201 @@
+name: AWS Commercial Canaries
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - "*"
+
+jobs:
+  upload:
+    runs-on: ubuntu-latest
+    # See https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token
+    permissions:
+      # required by aws-actions/configure-aws-credentials
+      id-token: write
+      contents: read
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '14'
```

Comment:
Could we use v16, 18 even?

Reply:
Thanks for pointing this out. Now using v18